www_src/pages/page/page.jsx

```diff
@@ -164,6 +166,14 @@ var Page = React.createClass({
           state.currentElementId = id;
         }
         this.setState(state);
+
+        //Ensure we don't reach a race conditions where the buttons are
+        //re-enabled before the menu is hidden. 200ms matches the css animation
+        //speed.
+
+        setTimeout(function() {
```

Comment:
the best place to start is to work through the "getting started" and "tutorial" sections on http://facebook.github.io/react, to get an understanding of how React wants you to structure things, and how to make use of the life cycle functions to trigger things. After that it depends a little on how much you picked up, but I'd be happy to help find further tutorials/resources on how to do specific things the "Reacty" way after running through React's own tutorial.